src/CloudFoundry.VisualStudio/Style/Controls.cs

```diff
@@ -12,17 +12,18 @@
  *
  * ***************************************************************************/
 
-using System;
-using System.Drawing;
-using System.Runtime.InteropServices;
-using System.Windows;
-using System.Windows.Interop;
-using System.Windows.Media.Imaging;
-using Microsoft.VisualStudio.PlatformUI;
-using Microsoft.VisualStudio.Shell;
-
 namespace CloudFoundry.VisualStudio.Style
 {
+    using System;
+    using System.Drawing;
+    using System.Runtime.InteropServices;
+    using System.Windows;
+    using System.Windows.Interop;
+    using System.Windows.Media.Imaging;
+    using Microsoft.VisualStudio.PlatformUI;
+    using Microsoft.VisualStudio.Shell;
+
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1724:TypeNamesShouldNotMatchNamespaces", Justification = "Styling necessary")]
```

Comment:
The justification here is incorrect.
We can safely rename this class to ControlColors.